examples/goldilocks_models/find_models/initial_guess_test.cc

```diff
@@ -32,16 +32,26 @@ int test_initial_guess(int iter, int sample, int robot) {
 
   const string dir =
       "../dairlib_data/goldilocks_models/find_models/robot_1_test/";
-  if (!CreateFolderIfNotExist(dir, false)) return 0;
```

Comment:
Why not use `CreateFolderIfNotExist`?

Reply:
Because I directly extract the parts used to create the folder from the `CreateFolderIfNotExist` and use it to create folder if it doesn't exist.